Code/Framework/AzCore/AzCore/Serialization/DataPatch.h

```diff
@@ -22,8 +22,8 @@ namespace AZ
         class AddressTypeSerializer;
 
         static constexpr const char PathDelimiter[] = "/";
-        static constexpr const char VersionDelimiter[] = u8"\u00B7"; // utf-8 for <middledot>
-        
+        static inline const char* VersionDelimiter = reinterpret_cast<const char*>(u8"\u00B7"); // utf-8 for <middledot>
```

Comment:
If this worked, we should consider changing the `PathDelimiter[]` line above to a `static inline const char*` as well

Reply:
Changed `VersionDelimiter` to `static inline const char*`